expressEngine/lib/errorHandler/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './errorHandler';
```

Comment:
Follow Airbnb guidelines for exporting from index

Reply:
corrected.